addons/mail/static/src/discuss/call/common/call_settings.scss

```diff
@@ -0,0 +1,23 @@
+// Define color variables for easy customization
+$filled-color: #0e9816;
```

Comment:
Variables in SCSS are in global scope, this name is too generic so can be reserved for some uses.
Also as it is it's not easily shareable: sometimes it works sometimes it doesn't, based on arbitrary ordering of loading SCSS files.

We solve it by defining variables in dedicate file, and bundle ensures this is loaded first. In mail this is `primary_variables.scss` & `core.scss`. There are subtleties between them which are more annoyances than anything... But better put there.

For the variable name, I suggest `$o-mail-X` where X is anything you can. I would encourage using somewhat a semblance of BEM naming convention, just so that it's easier to read and it looks nice and recognizable